crates/re_components/src/rect.rs

```diff
@@ -4,6 +4,8 @@ use arrow2_convert::{ArrowDeserialize, ArrowField, ArrowSerialize};
 
 use super::LegacyVec4D;
 
+/// TODO(#3243): Legacy component still used in some tests, benchmarks and examples.
+///
```

Comment:
we generally don't want TODO:s to show up in documentation

```suggestion
// TODO(#3243): Legacy component still used in some tests, benchmarks and examples.
//
```